sdk/nextjs/src/server/requests.ts

```diff
@@ -19,3 +22,56 @@ export const fetchCDNConfig = async (
         },
     )
 }
+
+const getSDKAPIUrl = (
+    sdkKey: string,
+    obfuscated: boolean,
+    user: DVCPopulatedUser,
+) => {
+    const searchParams = new URLSearchParams()
+    serializeUserSearchParams(user, searchParams)
+    searchParams.set('sdkKey', sdkKey)
+    if (obfuscated) {
+        searchParams.set('obfuscated', '1')
+    }
+    searchParams.set('sdkPlatform', 'nextjs')
+    searchParams.set('sse', '1')
+    return `https://sdk-api.devcycle.com/v1/sdkConfig?${searchParams.toString()}`
+}
+
+export const sdkConfigAPI = async (
+    sdkKey: string,
+    obfuscated: boolean,
+    user: DVCPopulatedUser,
+): Promise<Response> => {
+    return await fetch(getSDKAPIUrl(sdkKey, obfuscated, user), {
+        next: {
+            revalidate: 60,
+            tags: [sdkKey, user.user_id],
```

Comment:
so does this mean that the config request will be cached for the user regardless of if their user data changes? (aside from userid of course)

Reply:
It's only cached if the entire contents of the request are the same. So if the user data changes that will change the query params so it won't be cached